docs/start/_toc.json

```diff
@@ -10,7 +10,7 @@
       "title": "Install",
       "children": [
         {
-          "title": "Install and set up the Qiskit SDK",
+          "title": "Install Qiskit",
```

Comment:
@javabster fixed those with me -- it's intentional. The SDK part isn't necessary when referring to both Runtime and Qiskit install. And we removed the "set up" from the title because the setup actually happens on the channel setup page instead :)